Eatery/Controllers/BRB Feature/BRBViewController.swift

```diff
@@ -69,137 +94,274 @@ class BRBViewController: UIViewController, WKNavigationDelegate, BRBLoginViewDel
         }
     }
     
     func setupAccountPage() {
         
+        diningHistory = connectionHandler.diningHistory
+        
+        ai.transform = .identity
+        if diningHistory.count == 0 {
+            ai.startAnimating()
+        }
+        
         navigationItem.rightBarButtonItem?.isEnabled = true
         
-        let brbString = NSMutableAttributedString(string: "$\(connectionHandler.accountBalance.brbs)")
-        brbString.addAttributes([NSFontAttributeName: UIFont.systemFont(ofSize: 50.0)], range: NSRange(location: 0, length: 1))
-        brbString.addAttributes([NSFontAttributeName: UIFont.systemFont(ofSize: 50.0)], range: NSRange(location: brbString.length - 3, length: 3))
-        brbString.addAttributes([NSFontAttributeName: UIFont.systemFont(ofSize: 80.0)], range: NSRange(location: 1, length: brbString.length - 4))
-        
-        let brbLabel = UILabel(frame: CGRect(x: 0, y: view.frame.height * 0.15, width: view.frame.width, height: 120))
-        brbLabel.textColor = UIColor.black.withAlphaComponent(0.8)
-        brbLabel.attributedText = brbString
-        brbLabel.textAlignment = NSTextAlignment.center
-        view.addSubview(brbLabel)
-        
-        let brbDescriptionLabel = UILabel(frame: CGRect(x: 0, y: brbLabel.frame.origin.y + 85, width: view.frame.width, height: 50))
-        brbDescriptionLabel.textColor = UIColor.black.withAlphaComponent(0.8)
-        brbDescriptionLabel.text = "Big Red Bucks"
-        brbDescriptionLabel.font = UIFont.systemFont(ofSize: 20.0)
-        brbDescriptionLabel.textAlignment = NSTextAlignment.center
-        view.addSubview(brbDescriptionLabel)
-        
-        let swipesLabel = UILabel(frame: CGRect(x: 0, y: brbDescriptionLabel.frame.origin.y + brbDescriptionLabel.frame.height + 50, width: view.frame.width, height: 120))
-        swipesLabel.font = UIFont.systemFont(ofSize: 80.0)
-        swipesLabel.textColor = UIColor.black.withAlphaComponent(0.8)
-        swipesLabel.text = "\(connectionHandler.accountBalance.swipes)"
-        swipesLabel.textAlignment = NSTextAlignment.center
-        view.addSubview(swipesLabel)
-        
-        let swipesDescriptionLabel = UILabel(frame: CGRect(x: 0, y: swipesLabel.frame.origin.y + 85, width: view.frame.width, height: 50))
-        swipesDescriptionLabel.textColor = UIColor.black.withAlphaComponent(0.8)
-        swipesDescriptionLabel.text = "Swipes"
-        swipesDescriptionLabel.font = UIFont.systemFont(ofSize: 20.0)
-        swipesDescriptionLabel.textAlignment = NSTextAlignment.center
-        view.addSubview(swipesDescriptionLabel)
+        tableView = UITableView(frame: CGRect(x:0, y:0, width: (navigationController?.view.bounds.width ?? 0), height: (navigationController?.view.bounds.height ?? 0) - (navigationController?.navigationBar.frame.maxY ?? 0) - (tabBarController?.tabBar.frame.height ?? 0)), style: .plain)
+        tableView.register(BRBTableViewCell.self, forCellReuseIdentifier: "BalanceCell")
```

Comment:
Needs to be 3 different cell classes instead of trying to juggle this one class with multiple reuse identifiers